Security/src/AuthClient/Controllers/HomeController.cs

```diff
@@ -0,0 +1,118 @@
+using Microsoft.AspNetCore.Authentication;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+using Steeltoe.Samples.AuthClient.Models;
+using System;
+using System.Diagnostics;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Threading.Tasks;
+
+namespace Steeltoe.Samples.AuthClient.Controllers;
+
+public sealed class HomeController(IHttpClientFactory clientFactory, ILogger<HomeController> logger) : Controller
+{
+    public IActionResult Index()
+    {
+        return View();
+    }
+
+    #region SSO
+
+    [Authorize(Policy = Globals.RequiredJwtScope)]
+    public IActionResult TestGroup()
+    {
+        ViewData["Message"] = $"You have the '{Globals.RequiredJwtScope}' permission.";
+        return View();
+    }
+
+
+    [Authorize(Policy = Globals.UnknownJwtScope)]
+    public IActionResult AnotherTestGroup()
+    {
+        ViewData["Message"] = $"You have the '{Globals.UnknownJwtScope}' permission.";
+
+        return View("TestGroup");
+    }
+
+    [HttpGet]
+    [Authorize]
+    public IActionResult Login()
+    {
+        return RedirectToAction(nameof(Index), "Home");
+    }
+
+    public IActionResult Manage()
+    {
+        return View();
+    }
+
+    [HttpPost]
+    public async Task<IActionResult> LogOff()
+    {
+        await HttpContext.SignOutAsync();
+        return RedirectToAction(nameof(Index), "Home");
+    }
+
+    #endregion
+
+    [Authorize(Policy = Globals.RequiredJwtScope)]
+    public async Task<IActionResult> InvokeJwtSample()
+    {
+        using HttpClient jwtHttpClient = clientFactory.CreateClient("default");
+        string? token = await HttpContext.GetTokenAsync("access_token");
+        if (!string.IsNullOrEmpty(token))
+        {
+            jwtHttpClient.DefaultRequestHeaders.Authorization = new AuthenticationHeaderValue("Bearer", token);
+            return View("InvokeService", await SendRequestToBackend(jwtHttpClient, "/api/JwtAuthorization"));
```

Comment:
Please change these lines to capture the model in a separate variable, to make debugging easier. For example:
```c#
string model = await SendRequestToBackend(mutualTlsHttpClient, "/api/certificate/SameSpace");
return View("InvokeService", model);
```